functions/go/gatekeeper/go.mod

```diff
@@ -3,12 +3,12 @@ module github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/gateke
 go 1.17
 
 require (
-	github.com/open-policy-agent/frameworks/constraint v0.0.0-20211025234246-f478d8a555b5
-	github.com/open-policy-agent/gatekeeper v0.0.0-20211115204247-3ba8e93049b5 // This is v3.7.0. It has a semver major version of 2 or higher and is not a Go module yet.
-	github.com/spf13/cobra v1.1.3
-	k8s.io/apimachinery v0.21.2
+	github.com/open-policy-agent/frameworks/constraint v0.0.0-20220121182312-5d06dedcafb4
+	github.com/open-policy-agent/gatekeeper v0.0.0-20220208150435-b36e85531dbe // This is v3.7.0. It has a semver major version of 2 or higher and is not a Go module yet.
```

Comment:
I guess the version is no longer v3.7.0. Please update the comment.

Reply:
This is the current `HEAD` of gatekeeper, and doesn't correspond to a SEMVER tag.  What would you like in the comment?  Could we just remove it?

I think this is implied by the `go.mod` format of including the date and commit hash, rather than a SEMVER tag.

Reply:
Let's remove the comment here